### PR TITLE
doc: update todo app documentation

### PR DESCRIPTION
**Pull request by jayahariv**
* updated with 2.5 information. 
* add more platforms.

### CODE REVIEW

**Comment by pasin** [comment]
Just approve the PR and saw that might need to change the XCode version as well.

File: README.md [hunk 1]
```diff
@@ -1,36 +1,39 @@
...
+ToDo iOS sample app built with [CouchbaseLite 2.5.0](https://github.com/couchbase/couchbase-lite-ios/tree/release/iridium). 
+The application has Android, DotNet, Objective-C and Swift versions.
 
 ## Requirements
 - XCode 8.3
 
+
 ## How to build and run?
-1. Clone the project and checkout `feature/2.0` branch.
+1. Clone the project and checkout `release/iridium` branch.
```
Comment (jayahariv) [inline README.md]:
```suggestion
1. Clone the project and checkout `feature/2.5` branch.
```

File: README.md [hunk 2]
```diff
@@ -1,36 +1,39 @@
...
+1. Clone the project and checkout `feature/2.5` branch.
 
  ```
  $ git clone https://github.com/couchbaselabs/mobile-training-todo.git
- $ git checkout feature/2.0
+ $ git checkout feature/2.5
  ```
  
-2. cd into the project either `objc` or `swift`.
+2. cd into the project, `android`, `dotnet`, `objc` or `swift`.
```
Comment (jayahariv) [inline README.md]:
```suggestion
2. cd into the project: `android`, `dotnet`, `objc` or `swift`.
```

File: README.md [hunk 3]
```diff
@@ -1,36 +1,39 @@
 # ToDo Application
-ToDo iOS sample app built with [CouchbaseLite 2.0 Developer Preview](https://github.com/couchbase/couchbase-lite-ios/tree/feature/2.0). 
-The application has both Objective-C and Swift version.
+ToDo iOS sample app built with [CouchbaseLite 2.5.0](https://github.com/couchbase/couchbase-lite-ios/tree/release/iridium). 
+The application has Android, DotNet, Objective-C and Swift versions.
 
 ## Requirements
 - XCode 8.3
```
Comment (jayahariv) [inline README.md]:
```suggestion
- XCode 10.0
```

**Pull request merged**